config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadata.java

```diff
@@ -13,7 +13,7 @@
 import java.util.Collection;
 import java.util.LinkedList;
 
-@JsonPropertyOrder({"metaVersion", "id", "name", "groupName", "version", "type", "tags", "description", "reusable",
+@JsonPropertyOrder({"metaVersion", "id", "name", "groupName", "version", "type", "tags", "description", "portalTemplate", "reusable",
```

Comment:
There's a small discussion going on this field name (**portalTemplate**). Please let me know before you merge this PR.

Reply:
ok

Reply:
Please change the **portalTemplate** flag to **l7Template** as per the final METADATA spec:
https://cawiki.ca.com/display/Central/Developer+Plugin+Enhancements#DeveloperPluginEnhancements-BundleMetadata